docs/getting_started/trivia_storage.md

```diff
@@ -76,7 +76,7 @@ storeGlobal: async (state, event) => {
 
 ## Full database
 
-If nothing of the above works for you, you might want to consider using the built-in database (SQLite or Postgres). This is the most flexible option but obviously comes with more work on your end, as you'll have to manually create and maintain tables, rows and write SQL queries.
+If none of the above works for you, you might want to consider using one of the built-in databases (SQLite or Postgres). This is the most flexible option but obviously comes with more work on your end, as you'll have to manually create and maintain tables, rows and write SQL queries.
```

Comment:
There's only one built-in database, but they can change it (thus why I made it singular)

Reply:
You had a similar issue on the previous.  What you have written is grammatically incorrect, but I will amend to be closer to your meaning.